Task widget + widget header

prodin-ian opened the pull request:
fusion of previous WidgetHeader and TasksWidget PR's

saadnoor commented on widgets/DashboardTasks/DashboardTasks.css:
```diff
@@ -0,0 +1,89 @@
+    font-weight: var(--bbj-font-weight-lighter);
+    font-size: var(--bbj-font-size-xxs);
+    line-height: 16.4px;
+    color: var(--bbj-font-light, #A1A5A7);
```
can we remove hardcoded color?

saadnoor commented on widgets/DashboardTasks/DashboardTasksCard.bbj:
```diff
@@ -0,0 +1,78 @@
+        rightColumn!.setDockStyle("flex-basis", "80%")
+
+        #markComplete! = leftColumn!.addCheckBox(leftColumn!.getAvailableControlID(),0,0,0,0,"")
+        #markComplete!.setStyle("margin-top", "1.2rem")
```
remove this setstyle and move the code to a separate class.

saadnoor commented on widgets/DashboardWidgetHeader/DashboardWidgetHeader.bbj:
```diff
@@ -0,0 +1,103 @@
+                    dropdownText!.addStyle("widgetHeaderSettingDropdownItem")
+          
+
+                    dropdownTextBox2! = #settingDropdown!.addChildWindow(#settingDropdown!.getAvailableControlID(),0,0,0,0,"",$00100800$,BBjAPI().getSysGui().getAvailableContext())
```
use better variable names, insteadof dropdowntextbox1,2..

saadnoor commented on widgets/DashboardWidgetHeader/DashboardWidgetHeader.css:
```diff
@@ -0,0 +1,94 @@
+
+.widgetHeaderSettingDropdownPanel {
+    display:none;
+    position: absolute !important;
```
use position fixed. It'll break in new bbj version.

prodin-ian commented:
Addressed requested changes. Simplified WidgetHeader, right now its purely a graphical component. Needs further disccussion about required functionality
